client/app/components/land-use-action.hbs

```diff
@@ -0,0 +1,82 @@
+<h5>Proposed Land Use Actions</h5>
+<p>What land use actions does the proposed project require? Identify all that applyand fill in the appropriate related information. Note: This is preliminary information and may change following discussion with NYC Planning.</p>
```

Comment:
This content is already in the PasForm component. The LandUseAction component should just be the dropdown and dynamic inputs.

Reply:
You could also go ahead an plug this component into the pas-form/edit.hbs, swapping out lines 55-56: 
```
{{!-- TODO: Add land-use-actions-editor component --}}
<div class="bg-red-dim xlarge-padding medium-margin-bottom">land-use-actions-editor component</div>
```

Reply:
Okay good to know, I'll remove it!